packages/services/hmi-server/src/main/java/software/uncharted/terarium/hmiserver/proxies/mira/DKGProxy.java

```diff
@@ -0,0 +1,25 @@
+package software.uncharted.terarium.hmiserver.proxies.mira;
+
+import org.eclipse.microprofile.rest.client.inject.RegisterRestClient;
+import software.uncharted.terarium.hmiserver.models.mira.DKG;
+
+
+import javax.ws.rs.*;
+import javax.ws.rs.core.MediaType;
+import java.util.List;
+
+@RegisterRestClient(configKey = "mira-dkg")
+@Produces(MediaType.APPLICATION_JSON)
+public interface DKGProxy {
+	@GET
+	@Path("/api/entity/{curie}")
+	DKG getEntity(
+		@PathParam("curie") final String curie
+	);
+
```

Comment:
Also completely optional, but used elsewhere in our application.

If you annotation the interface with the common portions of the API paths (`@Path("/api")`) you can then annotate these methods with just the unique portion (for example, `@Path("/entities/{curies}")`).  Optional, but if your class had a lot of endpoints it makes it a bit easier to read